tools/bpf2c/bpf_code_generator.cpp

```diff
@@ -1215,6 +1220,16 @@ bpf_code_generator::emit_c_code(std::ostream& output_stream)
             if (program.program_info_hash.has_value()) {
                 output_stream << INDENT INDENT << program_info_hash_name << "," << std::endl;
                 output_stream << INDENT INDENT << program.program_info_hash.value().size() << "," << std::endl;
+                // Append the hash type
+                std::string hash_string = program.program_info_hash_type;
+                if (hash_string.empty()) {
+                    // If the hash type is not known, use the default hash type.
+                    // Current, only SHA256 is supported.
```

Comment:
```suggestion
                    // Current, only SHA-256 is supported.
```
While the "CNG Algorithm Identifier" is "SHA256", the algorithm is named SHA-256.